Gopkg.lock

```diff
@@ -10,24 +10,24 @@
   version = "v1.3.0"
 
 [[projects]]
-  digest = "1:ffe9824d294da03b391f44e1ae8281281b4afc1bdaa9588c9097785e3af10cec"
+  digest = "1:a2c1d0e43bd3baaa071d1b9ed72c27d78169b2b269f71c105ac4ba34b1be4a39"
   name = "github.com/davecgh/go-spew"
   packages = ["spew"]
   pruneopts = "UT"
-  revision = "8991bc29aa16c548c550c7ff78260e27b9ab7c73"
-  version = "v1.1.1"
+  revision = "346938d642f2ec3594ed81d874461961cd0faa76"
+  version = "v1.1.0"
```

Comment:
I see that we have versions decreased for multiple plugins. Do we have a reason for it?

Reply:
`dep` did that for some reason. Just updated vendors.

Reply:
`dep` did that for some reason. Just updated vendors.

Reply:
`dep` did that for some reason. Just updated vendors.